src/controllers/member.js

```diff
@@ -0,0 +1,59 @@
+const mongoose = require('mongoose');
+const Member = require('../models/Member');
+
+const getAllMembers = (req, res) => {
+  Member.find()
+    .then((member) => {
+      if (!member) {
+        return res.status(404).json({
+          message: 'Member not found',
```

Comment:
I think this message it's a little bit confusing because you ara trying to get all members, maybe it should say Members are empty or something like that

Reply:
You're right, I'll change the message